crates/ruma-client-api/src/sync/sync_events/v5.rs

```diff
@@ -549,8 +550,10 @@ pub mod response {
         /// index”. For example, consider `roomA` with `bump_stamp = 2`, `roomB`
         /// with `bump_stamp = 1` and `roomC` with `bump_stamp = 0`. If `roomC`
         /// receives an update, its `bump_stamp` will be 3.
+        ///
+        /// Note that this number can be negative.
```

Comment:
We should probably explain why it can be negative.

Reply:
Discussing with the Synapse team, it seems to be a bug on the server-side. But they are suggesting to still accept negative values client-side for the sake of robustness.

Reply:
I've improved the documentation with the current information I've.

Reply:
Synapse uses the stream ordering for the bump stamp, and for backfilled events it will be negative

It's unlikely that the client wants to use the negative bump stamp in this specific case, but since the MSC doesn't specify that it should be non-negative, this change seems reasonable anyway